analyzer/spec/functional/surface_spec_spec.rb

```diff
@@ -7,48 +7,77 @@ module Interpreter
       let(:concept) { Concepts::SurfaceSpec.new(:spec_name) }
       let(:spec) { Interpreter::SurfaceSpec.new(concept) }
 
-      before(:each) do
+      before do
         interpret_basis
+        spec.interpret('atoms c1: C%d, c2: C%d, c3: C%d')
       end
 
-      describe '#position' do
-        describe 'both atoms has lattice' do
-          let(:incomplete) { syntax_error('position.incomplete') }
+      describe 'default behavior for position instances' do
+        it { expect {
+            spec.interpret("position :c1, :c2, face: 100, dir: :front")
+          }.not_to raise_error }
```

Comment:
`end` at 18, 10 is not aligned with `expect {` at 16, 13 or `it { expect {` at 16, 8